datafusion/sqllogictest/test_files/expr.slt

```diff
@@ -1096,23 +1096,27 @@ SELECT date_part('nanosecond', timestamp '2020-09-08T12:00:12.12345678+00:00')
 ----
 12123456780
 
-query R
+# Second argument should not be string, failed in postgres too.
+query error
 SELECT date_part('second', '2020-09-08T12:00:12.12345678+00:00')
+
+query R
+SELECT date_part('second', timestamp '2020-09-08T12:00:12.12345678+00:00')
 ----
 12.12345678
 
 query R
-SELECT date_part('millisecond', '2020-09-08T12:00:12.12345678+00:00')
+SELECT date_part('millisecond', timestamp '2020-09-08T12:00:12.12345678+00:00')
```

Comment:
because the valid type should be timestamp but not string.

This is the result in Postgres
```
postgres=# SELECT date_part('millisecond', timestamp '2020-09-08T12:00:12.12345678+00:00');
 date_part 
-----------
 12123.457
(1 row)

postgres=# SELECT date_part('millisecond', '2020-09-08T12:00:12.12345678+00:00');
ERROR:  function date_part(unknown, unknown) is not unique
LINE 1: SELECT date_part('millisecond', '2020-09-08T12:00:12.1234567...
               ^
HINT:  Could not choose a best candidate function. You might need to add explicit type casts.
```